app/javascript/mastodon/features/ui/components/link_footer.js

```diff
@@ -60,31 +60,31 @@ class LinkFooter extends React.PureComponent {
           <Link key='about' to='/about'><FormattedMessage id='footer.about' defaultMessage='About' /></Link>
           {canInvite && (
             <>
-              {' · '}
+              <span aria-hidden>{' · '}</span>
               <a key='invites' href='/invites' target='_blank'><FormattedMessage id='footer.invite' defaultMessage='Invite people' /></a>
             </>
           )}
           {canProfileDirectory && (
             <>
-              {' · '}
+              <span aria-hidden>{' · '}</span>
               <Link key='directory' to='/directory'><FormattedMessage id='footer.directory' defaultMessage='Profiles directory' /></Link>
             </>
           )}
-          {' · '}
+          <span aria-hidden>{' · '}</span>
           <Link key='privacy-policy' to='/privacy-policy'><FormattedMessage id='footer.privacy_policy' defaultMessage='Privacy policy' /></Link>
         </p>
 
         <p>
           <strong>Mastodon</strong>:
-          {' '}
+          <span aria-hidden>{' · '}</span>
```

Comment:
This line is just white space. Is this change necessary?

Reply:
Nice catch! Fixed